parachain/pallets/ethereum-beacon-client/src/config.rs

```diff
@@ -36,4 +38,7 @@ pub const GENESIS_FORK_VERSION: ForkVersion = [30, 30, 30, 30];
 
 /// DomainType('0x07000000')
 /// https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/beacon-chain.md#domain-types
-pub const DOMAIN_SYNC_COMMITTEE: [u8; 4] = [7, 0, 0, 0];
\ No newline at end of file
+pub const DOMAIN_SYNC_COMMITTEE: [u8; 4] = [7, 0, 0, 0];
+
+pub const PUBKEY_SIZE: usize =  48;
+pub const SIGNATURE_SIZE: usize =  96;
```

Comment:
Nit: Missing newline at end of tile

Reply:
Fixed in https://github.com/Snowfork/snowbridge/commit/9b99699b7138ab58727203a687e05e08f740598f.